src/bundle.ts

```diff
@@ -95,3 +108,52 @@ export class BuildBundler extends Transform {
         });
   }
 }
+
+export type ResolveFileCallback = (a: string) => void;
+export type RejectFileCallback = (err: Error) => void;
+export type DeferredFileCallbacks = {
+  resolve: ResolveFileCallback; reject: RejectFileCallback;
+};
+
+class StreamLoader implements UrlLoader {
```

Comment:
This looks very similar to the class of the same name in src/analyzer.ts, what's up with the difference?

Document it in source?